techniques/general/G188.html

```diff
@@ -140,28 +136,28 @@ <h2>Related Resources</h2>
 
 
 <section id="related">
-		<h2>Related Techniques</h2>
+		<h2>関連テクニック</h2>
 		<ul>
       <li><a href="../css/C21">C21: Specifying line spacing in CSS</a></li>
       <li><a href="../css/C22">C22: Using CSS to control visual presentation of text</a></li>
       <li><a href="../css/C29">C29: Using a style switcher to provide a conforming alternate version</a></li>
    </ul>
 	</section>
 <section id="tests">
-			<h2>Tests</h2>
+			<h2>検証</h2>
       <section class="procedure" id="procedure">
-				<h3>Procedure</h3>
+				<h3>手順</h3>
 				<ol>
-					<li>Check that there is a button or link on the page that increases the size of the line height and the paragraph spacing, which is labeled as such.</li>
-          <li>Activate the button or link.</li>
-          <li>Check that the button or link increases the line height to at least 1.5 (150%)</li>
-          <li>Check that the button or link increases the paragraph spacing at least 1.5 times greater than the line spacing.</li>
+					<li>ページ内に行送り及び段落の間隔を広げるためのボタン又はリンクがあり、そのことがわかるようにラベルづけされている。</li>
```

Comment:
(もともとの翻訳の問題ですが) Check that ... を訳出しました。
```suggestion
					<li>ページ内に行送り及び段落の間隔を広げるためのボタン又はリンクがあり、そのことがわかるようにラベルづけされていることをチェックする。</li>
```